### PR TITLE
lower default max degree in riscv

### PR DESCRIPTION
**Pull request by leonardoalt**

### CODE REVIEW

File: riscv-executor/src/lib.rs [hunk 1]
```diff
@@ -724,12 +724,6 @@ mod builder {
 
             let main_degree = {
                 let range = namespace_degree_range(pil, "main");
-                assert!(
```
Comment (leonardoalt) [inline riscv-executor/src/lib.rs:727]: These assertions were too restrictive

Comment (leonardoalt) [reply riscv-executor/src/lib.rs:727]: and caused many_chunks to fail

Comment (chriseth) [inline riscv-executor/src/lib.rs:727]: Why can we remove those?

**Review by pacheco — APPROVED** [review]

File: riscv-executor/src/lib.rs (hunk 1 above)
Comment (pacheco) [reply riscv-executor/src/lib.rs:727]: for the continuations "dry run" it needs to run for the full execution (which will have a higher degree)

**Pull request merged**